data/human/culture conversations.txt

```diff
@@ -1071,6 +1071,150 @@ mission "Exotic Life: Geyser"
+				decline
+
+phrase "old man waiting"
+	word
+		"Brooding in the dim light,"
+		"Sitting with a weary stare,"
+		"Resting his weathered gaze,"
+		"Hunched over silent memories,"
+		"Lost in quiet rumination,"
+	word
+		" "
+	word
+		"his fingers tap the scarred wood,"
+		"his eyes drift over the shadowed floor,"
+		"his breath mingles with the smoky air,"
+		"his silence is smothered by the conversations of bar patrons,"
+	word
+		" "
+	word
+		"awaiting your next question with a secretive look."
+		"with mystery etched into every line of his face."
+		"poised at the edge of revelation and reticence."
+		"anticipating the spark of your next question."
+
+mission "Exotic Life: Albatross"
+	minor
+	source Albatross
+	to offer
+		random < 10
+	on offer
+		conversation
+			`You trudge through the biting wind to a weathered spaceport bar, its exterior marked by the signs proclaiming "No Gods, No Masters," and "No Cybernetic Implants. No Badgers, Weasels, Or Polecats." Inside, the air is heavy with shadow and smoke - worn wood panels, dim, flickering lights, and cluttered signs declaring even more peculiar prohibitions. Slouched in the corner by the heavy door, an old man sits silently, his half-lidded eyes watching the drunken banter and clinking glasses. You could approach him, and perhaps press for answers about the bar's odd proclamations.`
+			label nextquestion
+			`	${old man waiting}`
+			choice
+				`	"Why no Cybernetic implants?"`
+					goto implants
+					to display
+						not "heardimplants"
+				`	"What's the deal with Badgers?"`
+					goto badgers
+					to display
+						not "heardbadgers"
+				`	"Turnips?"`
+					to display
+						has "heardbadgers"
+						not "heardturnips"
+					goto turnips
+				`	"The sign says 'No ${plural nouns}'?"`
+					goto madlib
+					to display
+						or
+							has "heardimplants"
+							has "heardturnips"
+							has "heardweasels"
+						not "heardmadlib"
+				`	"What's wrong with Weasels?"`
+					goto weasels
+					to display
+						not "heardweasels"
+				`	"What about Polecats?"`
+					goto polecats
+					to display
+						has "heardweasels"
+						not "heardpolecats"
+				`	"Albatross?"`
+					goto albatross
+					to display
+						has "heardpolecats"
+				`	(Thank him for the time and leave.)`
+					goto ending
+			label implants
+			action
+				set "heardimplants"
+			`	The old man grunts, leaning forward so that his voice drops to a rough whisper. "Cybernetic impants ain't allowed 'round here - always cheat at cards. Can't trust a rigged hand when fate's supposed to be fair!"`
+				goto nextquestion
+			label badgers
+			action
+				set "heardbadgers"
+			`	He pauses, then mutters simply, "Turnips."`
+				goto nextquestion
+			label turnips
+			action
+				set "heardturnips"
+			`	He explodes into a brief tirade. "Badgers! They'll dig up every last turnip I grow, ruining my patch faster than a cyborg cleans house. Nothing but trouble!"`
```

Comment:
Naming what he "explodes into" is redundant when you directly show the player what he says.
```suggestion
			`	He explodes. "Badgers! They'll dig up every last turnip I grow, ruining my patch faster than a cyborg cleans house. Nothing but trouble!"`
```